pycon/settings.py

```diff
@@ -282,6 +282,11 @@
     # that's done this warning could be removed (and the order of the apps
     # should be no longer relevant)
 
+    # The conference app needs to be declared ahead of the `django` ones as
+    # it overrides some of django-admin commands. Moving other local apps here
+    # re-introduces deprecation warnings.
+    'conference',
```

Comment:
how that relates to the FIXME above? :)

Reply:
Well, the FIXME doesn't mention the `conference` app :) If the `conference` app has to follow `p3` and `assopy`, I can move them up here too; the only reason I haven't was because that introduces a bunch of Django 1.9 deprecation warnings.